docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.8'
+services:
+  blazor-app:
+    build:
+      context: .
+      dockerfile: ./Musakai.Blazor/Dockerfile.Blazor
```

Comment:
Is it necessary to have the exntesion .Blazor for the dockerfile?

Reply:
Nope, it was due to old docker-compose structure I used before rewriting